lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -187,10 +194,43 @@ export class PublicClientApplication extends ClientApplication implements IPubli
             });     
         } else {
             this.logger.verbose("acquireTokenSilent - attempting to acquire token from web flow");
+
             const silentCacheClient = this.createSilentCacheClient(request.correlationId);
             const silentRequest = await silentCacheClient.initializeSilentRequest(request, account);
-            result = silentCacheClient.acquireToken(silentRequest).catch(async () => {
-                return this.acquireTokenByRefreshToken(silentRequest);
+            
+            const requestWithCLP = {
+                ...request,
+                // set the request's CacheLookupPolicy to Default if it was not optionally passed in
+                cacheLookupPolicy: request.cacheLookupPolicy || CacheLookupPolicy.Default
+            };
+
+            result = this.acquireTokenFromCache(silentCacheClient, silentRequest, requestWithCLP).catch((cacheError: AuthError) => {
```

Comment:
@Robbie-Microsoft This looks like a breaking change for ExperimentalPublicClientApplication in msal-1p. Are we working on corresponding changes in 1p?
@jasonnutter